examples/react-todos/client/components/AppBody.jsx

```diff
@@ -19,24 +21,33 @@ setTimeout(function () {
   ShowConnectionIssues.set(true);
 }, CONNECTION_ISSUE_TIMEOUT);
 
+
+//
+// React classes.
```

Comment:
Not sure this comment is necessary, I feel like `React.createClass` is pretty self-explanatory